quasar-core/src/main/java/co/paralleluniverse/fibers/instrument/InstrumentClass.java

```diff
@@ -43,10 +43,14 @@
 
 import static co.paralleluniverse.fibers.instrument.Classes.*;
 import static co.paralleluniverse.fibers.instrument.QuasarInstrumentor.ASMAPI;
+
```

Comment:
Might be just a nitpick, but why blank lines between import statements?

Reply:
Visual grouping, lines above are "import static" lines below are straight "import".